src/DependencyInjection/Configuration.php

```diff
@@ -13,7 +13,7 @@ class Configuration implements ConfigurationInterface
     public function getConfigTreeBuilder(): TreeBuilder
     {
         $treeBuilder = new TreeBuilder('bedrock_rate_limit');
-        /** @var ArrayNodeDefinition $rootNode */
+        /** @var ArrayNodeDefinition $rootNode $rootNode */
```

Comment:
Looks like an error ?
```suggestion
        /** @var ArrayNodeDefinition $rootNode */
```

Reply:
done